plugin.rb

```diff
@@ -151,4 +151,12 @@ def create_json(obj)
     mount ::DiscourseStaffNotes::Engine, at: "/staff_notes"
   end
 
+  add_model_callback Warning, :after_create do
```

Comment:
Yes, this worked on my local test env.. maybe because of https://github.com/discourse/discourse/commit/db1820f39c0c019098d69d0c5a9b1d2f4056eb19?

Anyways, I will change it to `:warning`.